src/collapse/collapse.js

```diff
@@ -68,7 +68,7 @@ angular.module('mgcrea.ngStrap.collapse', [])
       self.$targets.$active = !self.$options.startCollapsed ? [0] : [];
       self.$setActive = $scope.$setActive = function(value) {
         if(angular.isArray(value)) {
-          self.$targets.$active = angular.copy(value);
+          self.$targets.$active = value;
```

Comment:
Not quite sure why there was ``` angular.copy ``` at the first place, but it adds extra difficulties to have persistent set of open panels if there is intermediate isolated scope (e.g. page controller -> custom conditional directive -> bs-collapse: custom directive will have all the changes, but page controller won't w/o extra watcher/formatter in custom directive scope)